src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -413,8 +413,13 @@ export default class AircraftCommander {
     runRoute(aircraft, data) {
         // TODO: is this .toUpperCase() necessary??
         const routeString = data[0].toUpperCase();
+        const readback = aircraft.pilot.applyPartialRouteAmendment(routeString);
 
-        return aircraft.pilot.applyPartialRouteAmendment(routeString);
+        if (readback[0] && aircraft.isDeparture()) {
+            aircraft.pilot.hasDepartureClearance = true;
+        }
```

Comment:
Please move this to happen in the pilot. The commander only exists to translate commands, and call the appropriate methods, and shouldn't be doing hardly anything in terms of modifying things.

Reply:
Because it doesn't really matter if it is a departure or not, so we don't really need that check at all.

Reply:
done